spec/presenters/scope_filters_presenter_spec.rb

```diff
@@ -50,90 +55,126 @@
+      presenter = described_class.new(paths: nil, organisation_slug: 'department-of-hats')
       expect(presenter).not_to be_invalid_filter
     end
 
-    it 'is false when both organisation and path are provided' do
-      presenter = described_class.new(path: '/done/buying-a-new-hat', organisation_slug: 'department-of-hats')
+    it 'is false when both `organisation_slug` and `paths` are provided' do
+      presenter = described_class.new(paths: ['/done/buying-a-new-hat'], organisation_slug: 'department-of-hats')
       expect(presenter).not_to be_invalid_filter
     end
   end
 
   describe "#done_page?" do
     it "is false when no path was provided" do
-      presenter = described_class.new(path: nil)
+      presenter = described_class.new(paths: nil)
       expect(presenter).not_to be_done_page
     end
 
-    it 'is true when the provided path starts with "/done"' do
-      presenter = described_class.new(path: "/done/buying-a-new-hat")
-      expect(presenter).to be_done_page
-    end
+    context "with one single path" do
+      it 'is true when the provided paths starts with "/done"' do
+        presenter = described_class.new(paths: ['/done/buying-a-new-hat'])
+        expect(presenter).to be_done_page
+      end
 
-    it 'is true when the provided path starts with "done"' do
-      presenter = described_class.new(path: "done/buying-a-new-hat")
-      expect(presenter).to be_done_page
-    end
+      it 'is true when the provided paths starts with "done"' do
+        presenter = described_class.new(paths: ['done/buying-a-new-hat'])
+        expect(presenter).to be_done_page
+      end
 
-    it 'is false when the provided path starts with anything else' do
-      presenter = described_class.new(path: "/start/buying-a-new-hat")
-      expect(presenter).not_to be_done_page
+      it 'is false when the provided paths starts with anything else' do
+        presenter = described_class.new(paths: ['/start/buying-a-new-hat'])
+        expect(presenter).not_to be_done_page
+      end
+
+      it 'is false when the provided paths has done somewhere other than the start' do
+        presenter = described_class.new(paths: ['/start/getting-things-done'])
+        expect(presenter).not_to be_done_page
+      end
     end
 
-    it 'is false when the provided path has done somewhere other than the start' do
-      presenter = described_class.new(path: "/start/getting-things-done")
-      expect(presenter).not_to be_done_page
+    context "with multiple paths" do
+      it 'is true when one of the provided paths starts with "/done"' do
+        presenter = described_class.new(paths: ['vat-rates', '/guidance/', '/done/buying-a-new-hat'])
+        expect(presenter).to be_done_page
+      end
+
+      it 'is true when one of the provided paths starts with "done"' do
+        presenter = described_class.new(paths: ['vat-rates', '/guidance/', '/done/buying-a-new-hat'])
+        expect(presenter).to be_done_page
+      end
```

Comment:
Minor: looks like this test is a duplicate of the preceding one?

Reply:
The preceding test checks for paths containing "/done", while this one checks for paths containing "done" without the forward slash. It's maybe a bit too thorough.

Reply:
So it does, I clearly wasn't paying enough attention.  It's good to be thorough!